src/ai/assistants/assistants.ts

```diff
@@ -126,6 +128,65 @@ export default class Assistants {
     return { ...txResult, assistant: assistant };
   }
 
+  async delete(
+    assistantId: string,
+    { config, tokenId }: AssistantDeleteParams
+  ): Promise<AssistantDeleteTransactionResult> {
+    const appId = Ainft721Object.getAppId(config.objectId);
+    const address = this.ain.signer.getAddress();
+
+    await validateObject(appId, this.ain);
+    await validateObjectOwnership(appId, address, this.ain);
+    await validateToken(appId, tokenId, this.ain);
+
+    const aiName = validateAndGetAiName(config.provider, config.api);
+    await validateAi(appId, aiName, this.ain);
+    await validateTokenAi(appId, tokenId, aiName, assistantId, this.ain);
+
+    const ai = await validateAndGetAiService(aiName, this.ainize);
+
+    // TODO(jiyoung): replace with ainize.request() function after deployment of ainize service.
+    // const response = await ai.request(<REQUEST_DATA>);
+    // NOTE(jiyoung): mocked deleted assistant.
+    const delAssistant = { id: assistantId, deleted: true };
+
+    const txBody = this.getAssistantDeleteTxBody(appId, tokenId, aiName);
+
+    const txResult = await this.ain.sendTransaction(txBody);
+    return { ...txResult, delAssistant };
+  }
+
+  async get(
+    assistantId: string,
+    { config, tokenId }: AssistantGetParams
```

Comment:
Reusing existing param types can make the get method a little bit complicated? How about just adding direct params instead? e.g.
```
  objectId: string,
  provider: string,
  api: string,
  tokenId: string,
```

Reply:
Done! this method is much clearer than before, thanks!